pkg/cmd/task/task_list.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const notAvailable = "N/A"
```

Comment:
nit: it sounds arbitrary to have the constant in this `dcos task list` while it is used in other commands as well. Maybe have it in the root task command.

Reply:
Fixed